scripts/exglobal_prep_ocean_obs.py

```diff
@@ -170,7 +170,16 @@ def run_netcdf_to_ioda(obsspace_to_convert):
     processes.append(process)
 
 # Wait for all processes to finish
+# TODO(AFE): add return value checking
 for process in processes:
     process.join()
 
-FileHandler({'copy': files_to_save}).sync()
+# TODO(AFE): Find a better way to do the "no file found" exception handling -
```

Comment:
Create an issue pointing to this line so we don't forget to disable this once the functionality is in FileHandler.

Reply:
done